src/core/server/saved_objects/service/lib/collect_multi_namespace_references.test.ts

```diff
@@ -445,4 +446,6 @@ describe('collectMultiNamespaceReferences', () => {
       );
     });
   });
+
+  test.todo('with security enabled');
```

Comment:
I added TODOs like this where unit tests need to be added to retain the code coverage that we had before this refactor.